File: Web-app-oppgave-2/Models/Rute.cs
using System.ComponentModel.DataAnnotations;

namespace Web_app_oppgave_2.Models
{
    public class Rute
    {
        [Key]
        public int RuteId { get; set; }
        public string Tur { get; set; }
        public byte[] Bilde { get; set; }
        public double Pris { get; set; }
    }
}

File: Web-app-oppgave-2/ClientApp/src/models/rute.ts
export class Rute{
  id?:number;
  tur:string;
  bilde?:any;
  pris:number;

  constructor(tur:string, pris:number) {
    this.tur = tur;
    //this.bilde = bilde;
    this.pris = pris;
  }

}
